### DISCUSSION TITLE
Re-enable developer tools and account sign in in Firefox

### DISCUSSION THREAD

**Discussion by risbi0** [simeononsecurity/Windows-Optimize-Harden-Debloat#77] [Q&A] [answered]
I'm a web developer and I need to access the devtools and I also prefer having my account signed in. Is there any way to re-enable those things?

**Comment by simeononsecurity** [comment, marked as answer]
@risbi0 

https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat#browsers
https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat/discussions/73

Be sure to specifically enable/disable the settings again. Do not set to "Not Configured".

**Reply by risbi0** [reply]
I downloaded `firefox.admx` and put it in `C:\Windows\PolicyDefinitions`, but I get an error popup when opening `gpedit.msc` which says: 

```
An appropriate resource file could not be found for file
C:\Windows\PolicyDefinitions\firefox.admx (error = 2): The
system cannot find the file specified.
```

EDIT: I just found your answer in a Q&A post and followed that.
